distribution/tools/plugin-cli/src/test/java/org/elasticsearch/plugins/cli/action/RemovePluginActionTests.java

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-package org.elasticsearch.plugins.cli;
+package org.elasticsearch.plugins.cli.action;
```

Comment:
I guess I wanted to be clearer about what classes were just for the CLI and what was shared with the server, but it's not as helpful as I hoped it would be. I'll move them back.